<launch>
    <arg name="num_controllers"            default="1"/>
    <arg name="serial_port_1"         default="/dev/for_docker/screw"/>
    <arg name="config_dir"            default="$(find o2as_fastening_tools)/config"/>
    <arg name="fastening_tools"       default="fastening_tools"/>

    <node name="dynamixel_controller" pkg="o2as_fastening_tools" type="dynamixel_controller" output="screen">
        <param name="num_controllers"      type="int"    value="$(arg num_controllers)"/>
        <param name="serial_port_1"   type="string" value="$(arg serial_port_1)"/>
    </node>

    <node name="fastening_tool_action_server" pkg="o2as_fastening_tools" type="fastening_tool_action_server.py" output="screen">
        <param name="config_dir"      type="string" value="$(arg config_dir)"/>
        <param name="fastening_tools" type="string" value="$(arg fastening_tools)"/>
    </node>
</launch>
